test/integration/node-specific/abort_signal.test.ts

```diff
@@ -942,7 +942,8 @@ describe('AbortSignal support', () => {
       const start = performance.now();
       const result = await collection.findOne({}, { signal }).catch(error => error);
```

Comment:
Bumping the timeout feels like the wrong approach here because presumably the root cause of the failure is that connection establishment took longer than we expected.  Can we consider instead pre-populating the pool with a connection?  If the test relies on the failCommand delay being the cause of the delay, then all other parts of the driver's execution should be configured to be as quick as possible.

We have waitUntilPoolsFilled in our test suite for this exact scenario.

Reply:
Feel free to make that change, sgtm, so you still want to expect it to finish in 10ms? maybe a bump and that change are worth it

Reply:
Okay, so I measured 100 iterations of this test with and without minPoolSize population.  pre-population seems to make no difference.  Here are some stats:

```
// no population
min:  0.7835420000010345
max:  3.556916999999885
average:  1.9274775899998895
```

```
// population
min:  0.5017919999954756
max:  3.9058329999998023
average:  1.8850562800002535
```

I also cannot reproduce the flakes locally.

Do you have any idea what is causing the delays we see in CI?  (i.e., which driver process, is it networking delays, etc)?  I don't want to just increase the timeout to "make the problem go away" without understanding why the test is flaking in the first place.

Reply:
I'm not sure, but we have evidence that CI is at least x2.3 slower than local development for the same code.  

Locally:
```
findManyAndEmptyCursor
    total time: 60.00434353699998 sec, ran: 453 times, median time per run: 0.13203516600000148 sec, taskSize: 16.22 mb, throughput: 122.84606057146789 mb/sec
```

In CI:
```
findManyAndEmptyCursor
  total time: 60.13917408399997 sec, ran: 200 times, median time per run: 0.30435943899999984 sec, taskSize: 16.22 mb, throughput: 53.292252257042726 mb/sec
```

I don't want to stop having the AbortSignal findOne coverage so I'm sure we can reconcile the two goals, should we split out the duration assertion?

Reply:
Oh, interesting.  That makes sense.  My M3 mac is even faster: 

```
  findManyAndEmptyCursor
    total time: 60.012349498 sec, ran: 570 times, median time per run: 0.10453212499999791 sec, taskSize: 16.22 mb, throughput: 155.16760995722916 mb/sec
```

That's a pretty substantial difference but I guess it would account for the flakiness - 3x3.5 or 3x4 (the `max` times I recorded locally) are above 10ms.

Reply:
So just the bump or do you still want to add the pool warming?